### PR TITLE
IceCreamMC

### PR DESCRIPTION
**Pull request by freezeoncrack**

### CODE REVIEW

**Review by Jerri1986 — APPROVED** [review]

File: _data/projects/icecreammc.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: IceCreamMC
+desc: A new minecraft server crafted with our new api plus love
+site: https://example.org/your-project-link-here
```
Comment (shiftkey) [inline _data/projects/icecreammc.yml:3]: This should be a valid project link

File: _data/projects/icecreammc.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+tags:
+  - java
+  - minecraft
+  - help wanted
+upforgrabs:
+  name: IceCreamMC
+  link: https://github.com/IceCreamMC
```
Comment (shiftkey) [inline _data/projects/icecreammc.yml:10]: This should be a valid repository link

**Review by shiftkey — COMMENTED** [review]
In it's current form this file doesn't point to any project. Unless changes are made, this will be closed out in a few days...

**Comment by shiftkey** [comment]
Closing out as inactive. Feel free to resubmit if the feedback above can be resolved...

**Pull request closed without merging**